.gitignore

```diff
@@ -4,6 +4,7 @@ node_modules
 pattern-lab
 source/_data/design-tokens.artifact.yml
 source/_patterns/00-config/_design-tokens.artifact.scss
+source/_patterns/_uswds-theme-settings.scss
```

Comment:
How do you feel about changing this filename to `_uswds-theme-settings.artifact.scss`?

Reply:
Good idea, that makes it more consistent with the other generated files. Will update.

Reply:
Updated!